### DISCUSSION TITLE
如何关闭Stack主题下markdown原生格式的图片缩放

### DISCUSSION THREAD

**Discussion by Siya-33** [CaiJimmy/hugo-theme-stack#870] [Q&A]
作者您好，由于markdown本身的图片调整不大方便，我目前是在正文中使用官方自带的`figure` shortcodes，但是它并不能使用Stack自带的缩放功能。于是我用了别的主题下的一种[解决办法](https://github.com/adityatelange/hugo-PaperMod/issues/384)，具体效果可以见[这里](https://www.russ.foo/2021/06/08/azure-devops-terraform-pipeline-with-checkov-approvals/)。主要操作是扩展了下`footer.html`

```
<script src="https://cdnjs.cloudflare.com/ajax/libs/medium-zoom/1.0.6/medium-zoom.min.js" integrity="sha512-N9IJRoc3LaP3NDoiGkcPa4gG94kapGpaA5Zq9/Dr04uf5TbLFU5q0o8AbRhLKUUlp8QFS2u7S+Yti0U7QtuZvQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>

<script>
const images = Array.from(document.querySelectorAll(".article-page .main-article .article-content img"));
images.forEach(img => {
  mediumZoom(img, {
    margin: 20, /* The space outside the zoomed image */
    scrollOffset: 40, /* The number of pixels to scroll to close the zoom */
    container: null, /* The viewport to render the zoom in */
    template: null, /* The template element to display on zoom */
    background: 'rgba(0, 0, 0, 0.8)'
  });
});
</script>


<!-- https://ionic.io/ionicons -->

<script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
<script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
```

但是这种情况下我再去使用`![](image.png)`这种格式时，两个功能会重叠，所以想问下有什么好一点的解决办法。我现在只能完全用figure替代了所有图片。

**Comment by CaiJimmy** [comment]
https://github.com/CaiJimmy/hugo-theme-stack/blob/c6d1d73dd21ff5e1c9c52db358e49857b3e0a6df/assets/ts/gallery.ts#L30-L31C19

Override this file and comment out `this.loadItems(container); this.bindClick();`

**Comment by Siya-33** [comment]
Thank you!

**Comment by Siya-33** [comment]
https://github.com/CaiJimmy/hugo-theme-stack/blob/c6d1d73dd21ff5e1c9c52db358e49857b3e0a6df/assets/ts/gallery.ts#L107

need to comment out this line,  otherwise it will open a new page when click
